File: 2nd_lab/3rd_task/3rd task.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Введите количество чисел: ");
        int n = scanner.nextInt();

        System.out.print("Введите числа через Enter: ");
        int count = 0;
        for (int i = 0; i < n; i++) {
            int num = scanner.nextInt();
            if (isPrime(num)) {
                count++;
            }
        }

        System.out.println("Количество простых чисел: " + count);
        scanner.close();
    }

    public static boolean isPrime(int num) {
        if (num <= 1) {
            return false;
        }
        for (int i = 2; i * i <= num; i++) {
            if (num % i == 0) {
                return false;
            }
        }
        return true;
    }
}

File: 2nd_lab/3rd_task/3rd task.kt
import kotlin.math.sqrt

// Функция для проверки простоты числа
fun isPrime(num: Int): Boolean {
    if (num <= 1) return false
    for (i in 2..sqrt(num.toDouble()).toInt()) {
        if (num % i == 0) return false
    }
    return true
}

// Функция для подсчета количества простых чисел
fun countPrimes(n: Int): Int {
    var count = 0
    print("Введите числа через Enter:\n")
    for (i in 1..n) {
        val num = readLine()?.toIntOrNull() ?: 0
        if (isPrime(num)) count++
    }
    return count
}

fun main() {
    print("Введите количество чисел: ")
    val n = readLine()?.toIntOrNull() ?: 0
    val result = countPrimes(n)
    println("Количество простых чисел: $result")
}
